SnakeGame/view.py

```diff
@@ -1,14 +1,49 @@
 import pygame
+from model import CellContent
+import enum
 
 class GameView(object):
     
-    DEFAULT_SIZE = width, length = 540, 540
     BG_COLOR = green = 0, 135, 0
+    SNAKE_COLOUR = 135, 0, 0
+    FOOD_COLOUR = 255, 51, 0
+    SNAKEHEAD_COLOR = 0, 0, 0 
 
-    def __init__(self):
-        self.size = self.DEFAULT_SIZE
+    def __init__(self, gridDimensions=(10, 10), gridSize=(540, 540)):
```

Comment:
I think gridDimensions should be a required argument here with no default value. The reason for that is that grid dimensions are entirely determined by the model. 

1. if we use a custom grid dimension value in the model and we forget to pass it here as well, we will get a weird mismatch.
2. if we change the default dimension in the model, we would have to change it here as well. Otherwise, we get the weird mismatch.

If the argument was required, we would _always_ do `view = GameView(model.gridSize)` in the controller.

I would also recommend changing gridSize to windowSize to avoid confusion (a find and replace should be good) but this isn't super important.